app/src/ui/copy-button.tsx

```diff
@@ -0,0 +1,71 @@
+import { clipboard } from 'electron'
+import React from 'react'
+import * as OcticonSymbol from './octicons/octicons.generated'
+import { Octicon } from './octicons'
+import { sleep } from '../lib/promise'
+import { Button } from './lib/button'
+import { AriaLiveContainer } from './accessibility/aria-live-container'
+
+interface ICopyButtonProps {
+  readonly copyContent: string
+  readonly ariaLabel: string
+}
+
+interface ICopyButtonState {
+  readonly showCopied: boolean
+}
+
+export class CopyButton extends React.Component<
+  ICopyButtonProps,
+  ICopyButtonState
+> {
+  public constructor(props: ICopyButtonProps) {
+    super(props)
+
+    this.state = {
+      showCopied: false,
+    }
+  }
+
+  private onCopy = async (e: React.MouseEvent<HTMLButtonElement>) => {
+    e.preventDefault()
+    clipboard.writeText(this.props.copyContent)
+
+    this.setState({ showCopied: true })
+
+    await sleep(2000)
+
+    this.setState({ showCopied: false })
+  }
+
+  public renderSymbol() {
+    const { showCopied } = this.state
+
+    const symbol = showCopied ? OcticonSymbol.check : OcticonSymbol.copy
+
+    return <Octicon symbol={symbol} />
+  }
+
+  public render() {
+    const { ariaLabel } = this.props
+    const { showCopied } = this.state
+
+    const copiedMessage = 'Copied!'
+    const ariaMessage = showCopied ? copiedMessage : ''
+    return (
+      <Button
+        className="copy-button"
+        tooltip={showCopied ? copiedMessage : ariaLabel}
+        ariaLabel={ariaLabel}
+        onClick={this.onCopy}
+        openTooltipOnClick={true}
+      >
+        {this.renderSymbol()}
+        <AriaLiveContainer
+          message={copiedMessage}
+          trackedUserInput={ariaMessage}
+        />
```

Comment:
Using this to announce the `Copied` message. Other thought was to update the button `ariaLabel` but that didn't seem to reliably announce. Could have gone further down that path of unfocusing then refocusing.. or something I thought of later and didn't try was adding a `sr-only` span with "copied" to the contents. But in the testing I did with the `AriaLiveContainer`, it seems to do the trick.